scripts/misc/download_model.py

```diff
@@ -109,7 +116,8 @@ def parse_args() -> argparse.Namespace:
         download_from_hf_hub(args.model,
                              save_dir=args.save_dir,
                              token=args.token,
+                             tokenizers_only=args.tokenizer_only,
```

Comment:
```suggestion
                             tokenizer_only=args.tokenizer_only,
```
For consistency with singular `tokenizer_only`